app/assets/javascripts/discourse/app/lib/parse-mentions.js

```diff
@@ -0,0 +1,9 @@
+export function parseMentionedUsernames(cooked) {
+  const html = new DOMParser().parseFromString(cooked, "text/html");
```

Comment:
Is this every going to be used on the server-side? I think we can't do `DOMParser` there, right?

Reply:
This makes sense, and it's fixed now. We don't use `DOMParser` anymore (it was fixed in main in 73781c8a96e35062b2631263bd6afd37f7f7016b and then merged into this branch).